nautobot/core/templates/panel_table.html

```diff
@@ -1,9 +1,9 @@
 {% load render_table from django_tables2 %}
 {% load helpers %}
 
-<div class="panel panel-{{ panel_class|default:'default' }}">
+<div class="card">
```

Comment:
Can/should we retain support for the `panel_class` attribute?

Reply:
Reading the card documentation [here](https://getbootstrap.com/docs/5.0/components/card/). Seems like `panel-*` e.g `panel-default` support is completely removed in Bootstrap 5. As I found no references on similar features.

Reply:
https://github.com/nautobot/nautobot/pull/7093/commits/26c6fbba2a111f793d08044aaa1a1afb288778f9